Marlin/src/inc/SanityCheck.h

```diff
@@ -4503,6 +4503,11 @@ static_assert(WITHIN(MULTISTEPPING_LIMIT, 1, 128) && IS_POWER_OF_2(MULTISTEPPING
   #error "Only enable ULTIPANEL_FEEDMULTIPLY or ULTIPANEL_FLOWPERCENT, but not both."
 #endif
 
+// Differential Extruder requires the magic of Creality V4
+#if ENABLED(DIFFERENTIAL_EXTRUDER) && !(EXTRUDERS == 1 && MB(CREALITY_V4))
+  #error "DIFFERENTIAL_EXTRUDER requires MOTHERBOARD BOARD_CREALITY_V4 and EXTRUDERS 1."
+#endif
```

Comment:
It looks to me like the feature could work with any board, including AVR, as there are no complex calculations.